From 7c1d52b6bda92626af537c26ae1e376171babe0d Mon Sep 17 00:00:00 2001
From: Valentin Raevsky <valentin@compulab.co.il>
Date: Tue, 29 Dec 2015 13:37:12 +0200
Subject: [PATCH 14/38] ARM: i.MX6UL: update cl-som-imx6ul defconfig for
 systemd

Update cl-som-imx6ul defconfig for systemd.
Source: cgit.freedesktop.org/systemd/systemd/tree/README

Signed-off-by: Valentin Raevsky <valentin@compulab.co.il>
---
 arch/arm/configs/cl_som_imx6ul_defconfig | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/arch/arm/configs/cl_som_imx6ul_defconfig b/arch/arm/configs/cl_som_imx6ul_defconfig
index 486b9a6..a658e70 100644
--- a/arch/arm/configs/cl_som_imx6ul_defconfig
+++ b/arch/arm/configs/cl_som_imx6ul_defconfig
@@ -8,6 +8,10 @@ CONFIG_IKCONFIG=y
 CONFIG_IKCONFIG_PROC=y
 CONFIG_LOG_BUF_SHIFT=18
 CONFIG_CGROUPS=y
+CONFIG_CGROUP_SCHED=y
+CONFIG_CFS_BANDWIDTH=y
+CONFIG_CHECKPOINT_RESTORE=y
+CONFIG_NAMESPACES=y
 CONFIG_RELAY=y
 CONFIG_BLK_DEV_INITRD=y
 CONFIG_EXPERT=y
@@ -40,6 +44,7 @@ CONFIG_PREEMPT=y
 CONFIG_AEABI=y
 CONFIG_HIGHMEM=y
 CONFIG_CMA=y
+CONFIG_SECCOMP=y
 CONFIG_CMDLINE="noinitrd console=ttymxc0,115200"
 CONFIG_CPU_FREQ=y
 CONFIG_CPU_FREQ_DEFAULT_GOV_INTERACTIVE=y
@@ -72,6 +77,7 @@ CONFIG_NF_CONNTRACK=y
 CONFIG_NF_CONNTRACK_IPV4=y
 CONFIG_IP_NF_IPTABLES=y
 CONFIG_IP_NF_FILTER=y
+CONFIG_IP6_NF_IPTABLES=y
 CONFIG_VLAN_8021Q=y
 CONFIG_LLC2=y
 CONFIG_CAN=y
@@ -99,6 +105,7 @@ CONFIG_MAC80211=y
 CONFIG_DEVTMPFS=y
 CONFIG_DEVTMPFS_MOUNT=y
 # CONFIG_STANDALONE is not set
+# CONFIG_FW_LOADER_USER_HELPER is not set
 CONFIG_DMA_CMA=y
 CONFIG_CMA_SIZE_MBYTES=320
 CONFIG_IMX_WEIM=y
@@ -208,6 +215,7 @@ CONFIG_INPUT_ISL29023=y
 CONFIG_INPUT_MPL3115=y
 CONFIG_SENSOR_FXLS8471=y
 CONFIG_SERIO_SERPORT=m
+CONFIG_DEVPTS_MULTIPLE_INSTANCES=y
 # CONFIG_LEGACY_PTYS is not set
 # CONFIG_DEVKMEM is not set
 CONFIG_SERIAL_IMX=y
@@ -419,6 +427,7 @@ CONFIG_UDF_FS=m
 CONFIG_MSDOS_FS=m
 CONFIG_VFAT_FS=y
 CONFIG_TMPFS=y
+CONFIG_TMPFS_XATTR=y
 CONFIG_JFFS2_FS=y
 CONFIG_UBIFS_FS=y
 CONFIG_NFS_FS=y
-- 
1.9.1

